ToDoTime/ClientApp/src/shared/BoxCustomIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { IconBaseProps } from "react-icons/lib";
+import * as BiIcons from "react-icons/bi";
+
+interface BoxCustomIconProps {
+  nameIcon: string;
+  propsIcon?: IconBaseProps;
+}
+
+export const BoxCustomIcon = ({ nameIcon, propsIcon }: BoxCustomIconProps) => {
+  // @ts-expect-error - For some reason ts doesnt understand this, but it works
+  const ElementIcon = BiIcons[nameIcon];
```

Comment:
This should consider returning a default icon in case the icon searched doesnt exist, probably good to add a test that  ensure this.

Reply:
Fixed and added test for it